evalml/tests/automl_tests/test_automl.py

```diff
@@ -1530,7 +1530,7 @@ def test_results_getter(AutoMLTestEnv, X_y_binary):
 
     assert automl.results["pipeline_results"][0]["mean_cv_score"] == 1.0
 
-    with pytest.raises(AttributeError, match="set attribute"):
+    with pytest.raises(AttributeError):
```

Comment:
Do you know why the changes lines in this file and in `test_pipeline_search_plots.py` are showing as uncovered by tests? That seems a bit strange to me.